authors/apps/authentication/models.py

```diff
@@ -95,19 +100,18 @@ class User(AbstractBaseUser, PermissionsMixin):
     def __str__(self):
         """
         Returns a string representation of this `User`.
-
         This string is used when a `User` is printed in the console.
         """
         return self.email
 
     @property
     def get_full_name(self):
-      """
-      This method is required by Django for things like handling emails.
-      Typically, this would be the user's first and last name. Since we do
-      not store the user's real name, we return their username instead.
-      """
-      return self.username
+        """
```

Comment:
@mwinel Avoid such unnecessary changes.

Reply:
@archibishop there was an indentation error here that was making the app fail to run.